client/src/panels/panel_declarations/results/results_intro_text.yaml

```diff
@@ -38,11 +38,11 @@ gba_plus_intro_text:
   transform: [handlebars,markdown]
   en: |
     The federal government uses {{gl_tt "gender-based analysis plus" "GBA_PLUS"}} (GBA Plus) in the development of policies, programs and legislation.
-    Indicators from the most recent results reports, **2019-20**, have been marked as {{gl_tt "supporting GBA Plus" "GBA_PLUS_INDICATOR"}} where applicable.
+    Indicators from the most recent results reports, **2019-20**, have been marked as {{gl_tt "GBA Plus" "GBA_PLUS_INDICATOR"}} where applicable.
     These indicators and their GBA Plus markings can be found on the pages for individual organizations.
   fr: |
     L'{{gl_tt "analyse comparative entre les sexes plus" "GBA_PLUS"}} (ACS Plus) est utilisée par le gouvernement fédéral pour l'élaboration de politiques, de programmes et de lois. 
-    Dans les plus récents rapports sur les résultats (**2019-2020**), les indicateurs ont été marqués comme {{gl_tt "appuyant l'ACS Plus" "GBA_PLUS_INDICATOR"}}, le cas échéant. 
+    Dans les plus récents rapports sur les résultats (**2019-2020**), les indicateurs portent un marqueur {{gl_tt "ACS Plus" "GBA_PLUS_INDICATOR"}}, le cas échéant. 
```

Comment:
@sranc046 Could you take a look and see if this sentence makes sense? All I'm trying to do is remove `supporting`

And just below this line on 46 `Ces indicateurs et **leur** marquage ACS Plus`, Does `leur` supposed to be `le`?